lib/tasks/build.js

```diff
@@ -3,13 +3,12 @@
 const path = require('path');
 const merge = require('merge-stream');
 const BowerPlugin = require('bower-webpack-plugin');
-const webpack = require('webpack-stream');
+const webpack = require('webpack');
+const webpackStream = require('webpack-stream');
```

Comment:
I don't know where webpack is being used, maybe it can be removed from the package.json and that's a mistake. Having it in the package.json slows down install, but having it here slows down the actual build